src/main.rs

```diff
@@ -356,20 +367,30 @@ fn main() {
     }
 }
 
-fn exec(cmd: Command) -> Result<String> {
+fn exec(cmd: Command) -> Result<Option<String>> {
     match &cmd {
         Command::New { name, target_dir } => cmd::new::execute(name, target_dir.as_ref()),
         Command::Build(build) => {
             let result = build.exec()?;
-            Ok(result.display())
+            if util::is_verbose(&result.verbosity) {
```

Comment:
Not nice to have to have these conditional blocks everywhere.

We could either replace the success status message propogation, and just call our `println!` ersatz macro directly like we would do in other places, or rework the args so the verbosity flags are available at the top level command.